modules/ROOT/pages/proxy-domain-support.adoc

```diff
@@ -15,16 +15,15 @@ API Manager enables you to automatically configure an API proxy to inherit an ex
 
 Note that API domains are not supported by Runtime Fabric or CloudHub support API domains. 
 
-*Important:* In an API domain, you may define several dependencies. Ensure that the versions of the dependencies defined in the domain have the same or later version than the ones defined in the API proxy that references the domain.
+*Important:* You may define several dependencies in an API domain. Ensure that the versions of the dependencies defined in the domain have the same or later version than the ones defined in the API proxy that references the domain.
 
-All the dependencies defined in the domain override the dependencies defined in the API proxy (older or newer). For example, if the API proxy depends on a functionality defined in the HTTP transport, but is referencing a domain using
- an older version of that transport, you will have errors at runtime.
+All the dependencies defined in the domain override the dependencies defined in the API proxy (older or newer). For example, if the API proxy depends on a functionality defined in the HTTP transport, but is referencing a domain using an older version of that transport, you will have errors at runtime.
```

Comment:
"you will have errors" should remove future tense and 2nd person pronoun. Something like "errors are thrown at runtime" or "the proxy throws errors at runtime".

Reply:
Done! Thanks!